### ISSUE TITLE
Godot 4.0  Alpha 6 Exporting bug

### ISSUE THREAD

**Issue by WouterB15** [godotengine/godot#60382]
### Godot version

4.0 Alpha 6

### System information

Windows 10, Vulkan, RTX 3070, Ryzen 5 3600

### Issue description

Everything worked fine in the editor, but when I exported it it all went wrong. I had some SVG images in use, which it didn't like. Replaced them for png's, which allowed me to launch, but it still doesn't behave as expected. Got these errors:

```
Godot Engine v4.0.alpha6.official.e4f0fc50f - https://godotengine.org
Vulkan API 1.2.182 - Using Vulkan Device #0: NVIDIA - NVIDIA GeForce RTX 3070

ERROR: Parser bug (please report): tried to assign unset node without an identifier.
   at: resolve_class_interface (modules/gdscript/gdscript_analyzer.cpp:656)
ERROR: Parser bug (please report): tried to assign unset node without an identifier.
   at: resolve_class_interface (modules/gdscript/gdscript_analyzer.cpp:656)
ERROR: Parser bug (please report): tried to assign unset node without an identifier.
   at: resolve_class_interface (modules/gdscript/gdscript_analyzer.cpp:656)
ERROR: Parser bug (please report): tried to assign unset node without an identifier.
   at: resolve_class_interface (modules/gdscript/gdscript_analyzer.cpp:656)
SCRIPT ERROR: Parse Error: Preload file "res://Images/succes.png" does not exist.
          at: GDScript::reload (res://Scripts/AlertPopup.gd:19)
SCRIPT ERROR: Parse Error: Preload file "res://Images/warning.png" does not exist.
          at: GDScript::reload (res://Scripts/AlertPopup.gd:20)
SCRIPT ERROR: Parse Error: Preload file "res://Images/error.png" does not exist.
          at: GDScript::reload (res://Scripts/AlertPopup.gd:21)
SCRIPT ERROR: Parse Error: Preload file "res://Images/info.png" does not exist.
          at: GDScript::reload (res://Scripts/AlertPopup.gd:22)
ERROR: Method/function failed. Returning: ERR_PARSE_ERROR
   at: reload (modules/gdscript/gdscript.cpp:870)
ERROR: Error calling from signal 'button_up' to callable: PanelContainer(AlertPopup.gd)::_on_close_button_up : Method not found..
   at: emit_signalp (core/object/object.cpp:1132)

```
The images that it says that don't exist do exist.
Also the method it says that doesn't exist does exist too.

### Steps to reproduce

~~Sadly haven't found a _consistent_ way to reproduce it.
Only thing that's off is that it's the only TextureButton I've got.~~

Using `preload()`

Edit: Managed to isolate the scene that had the issue, added it with the reproduction project

### Minimal reproduction project
[ExportError.zip](https://github.com/godotengine/godot/files/8522314/ExportError.zip)

**Comment by Calinou** [comment]
@WouterB15 Please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

**Comment by WouterB15** [comment]
@Calinou Sorry for late reply, went straight to bed after posting this. Added a reproduction project!

**Comment by ItsNL** [comment]
I'm having the same problem. When the project starts I have a node that preloads some sounds. When I export the game the preloaded sounds are gone. In the editor they work perfect, but when the game is exported they don't.
I tried changing from preload to load and it works great.

This doesn't work when exported:
`var bullet_fx = preload("res://path")`
But this does:
`var bullet_fx = load("res://path")`

I also noticed that when I open the game with the .cmd file the game crashes when the sound is required. But, when  it is a debug build and it is opened by the .exe file the game doesn't crash but the sound is not there.


**Version:** v4.0.alpha14.official [106b68050]
Windows 11
Vulkan API 1.2.0
